src/javaRosa.js

```diff
@@ -951,20 +948,18 @@ define([
                 widget.setValue(defaultValue);
                 widget.handleChange();
             } else {
-                var itextItem = widget.getItextItem();
+                var value = widget.getItextValue();
 
-                if (!itextItem) {
-                    widget.setValue("");
-                    return;
+                if (!_.isString(value)) {
```

Comment:
It's undefined for constraint, hint and help messages for a new mug. https://github.com/dimagi/Vellum/blob/bec373ab8ac1ceb586ed6c05a377a11289de8ead/src/javaRosa.js#L329-L342.